### DISCUSSION TITLE
DISA STIG released for Sonoma!!

### DISCUSSION THREAD

**Discussion by lucasbarmy** [usnistgov/macos_security#349] [General] [answered]
Hey, Team!  The DISA STIGs for Sonoma have dropped!  How long until the scripts under the project are updated for DISA Sonoma?  I know it just happened last week, but I don't know the usual lead time for the updates here in Github.

**Comment by jmahlman** [comment, marked as answer]
We'll look to get this into mSCP starting next week.

**Reply by lucasbarmy** [reply]
Thank you!

**Comment by robertgendler** [comment]
There's also a DISA stig dev branch already. It's just missing the STIG IDs

**Reply by lucasbarmy** [reply]
Nice, I'll take a look.
